wasm/src/debug_service.rs

```diff
@@ -195,3 +313,43 @@ struct StackFrame {
     pub lo: u32,
     pub hi: u32,
 }
+
+#[wasm_bindgen(typescript_custom_section)]
+const IVariableList: &'static str = r#"
+export interface IVariable {
+    name: string;
+    value: string;
+    var_type: "Array"
+        | "BigInt"
+        | "Bool"
+        | "Closure"
+        | "Double"
+        | "Global"
+        | "Int"
+        | "Pauli"
+        | "Qubit"
+        | "Range"
+        | "Result"
+        | "String"
+        | "Tuple";
+}
```

Comment:
Is there any issue with UDTs here?

Reply:
They show up as Tuple. We can enhance this later. Stefan and I tried some stuff but the perf hit was significant.